src/utils/utils.ts

```diff
@@ -76,3 +76,11 @@ export const getPageMode = (): PageMode => {
 
   return 'VIEW';
 };
+
+export const getAdminAPI = (): string => {
```

Comment:
duplicate data origin for Admin API & Key, please keep getting api & key from config file.

Reply:
Just keep it as simple as possible :)

Reply:
But this will cause every file used to be imported in the form of `import xxx from '../../config'` 
The `config.ts` is not in `src` fold. It can't be imported in `import xxx from '@/config.ts'` 
Define an function can help us to get configuration from one place and change it easy.

> In addition, constants in configuration files such as apikey are now automatically replaced at compile time by a umi plugin. That means we don't even need to import configuration files.

Reply:
~how about adding alias for config file? so we could import it like '@config'? ~

all right, please go ahead,

Reply:
> like `@/utils/config`.  But, `/config` is compile time configuration, `/src/xxxx` is runtime.
> 
> In fact, I use DefinePlugin by webpack, [description link](https://webpack.js.org/plugins/define-plugin/).
> First, declare it in the configuration file, `{ADMIN_API: 'xxxxx'}`, it will automatically replace the part of the program that uses the `ADMIN_API` at compile time without import, Such as
> `let adminAPI = ADMIN_API` will be compiled to `let adminAPI = 'xxxxx'`.

> write this function `getAdminAPI()`, because it's used many times, in order to reduce redundant code, I think it's best to do so.

Reply:
ok, take your opinion :)